mingw-w64-aws-sdk-cpp/PKGBUILD

```
@@ -0,0 +1,56 @@
+# Maintainer: Andrew Sun <adsun701@gmail.com>
+
+_realname=aws-sdk-cpp
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=1.7.365
+pkgrel=1
+pkgdesc="AWS SDK for C++ (mingw-w64)"
+arch=('any')
+url="https://github.com/aws/aws-sdk-cpp"
+license=('Apache-2.0')
+makedepends=("${MINGW_PACKAGE_PREFIX}-cmake")
+options=('strip' 'staticlibs')
+source=("${_realname}-${pkgver}.tar.gz::https://github.com/aws/aws-sdk-cpp/archive/${pkgver}.tar.gz"
+        "aws-sdk-cpp-pr-1333.patch"
+        "WinHttp-missing-flags.patch"
+        "Patch-cmake-submodules.patch"
+        "BuildAwsCCommon.patch"
+        "BuildAwsChecksums.patch")
+sha256sums=('95e3f40efaea7b232741bfb76c54c9507c02631edfc198720b0e84be0ebb5e9d'
+            'SKIP'
+            'SKIP'
+            'SKIP'
+            'SKIP'
+            'SKIP')
+
+prepare() {
+  cd "${_realname}-${pkgver}"
+  patch -p1 -i "${srcdir}/aws-sdk-cpp-pr-1333.patch"
+  patch -p1 -i "${srcdir}/WinHttp-missing-flags.patch"
+  patch -p1 -i "${srcdir}/Patch-cmake-submodules.patch"
+}
+
+build() {
+  export CFLAGS="-D_WIN32_WINNT=0x0600"
+  export CXXFLAGS="-D_WIN32_WINNT=0x0600"
+
+  [[ -d "${srcdir}"/build-${CARCH}-static ]] && rm -rf "${srcdir}"/build-${CARCH}-static
+  mkdir -p "${srcdir}"/build-${CARCH}-static && cd "${srcdir}"/build-${CARCH}-static
+  MSYS2_ARG_CONV_EXCL="-DCMAKE_INSTALL_PREFIX=;-DCMAKE_PREFIX_PATH=" \
+    ${MINGW_PREFIX}/bin/cmake \
+      -G'MSYS Makefiles' \
```

```suggestion
      -GNinja \
```
I recommend using the Ninja generator instead since MSYS Makefiles is more designed for mingw's msys, and not msys2 and also it's much faster overall

changing this will require adding ninja to the builddeps and the make at the bottom to ninja, but alternatively the make like can be changed to
```
cmake --build .
``` 
instead

---

Thanks! I did not know this. I have changed it to ninja now.